uwblueprint/don-efficace
# Add total value donated component

[create “Total Value Donated” component](https://www.notion.so/uwblueprintexecs/create-Total-Value-Donated-component-2cbe6eea519541a98ad8041e35c5cd89?pvs=4)



## Implementation description
* A new TotalValueDonated component has been added to the layout page temporarily
* The fetchTotalDonations function currently displays a static value in the component, but that can be refactored once we figure out how to be able to determine which user is logged in




## Steps to test
1. Visit http://localhost:3000/layout to verify that the manually inputted data is displayed in the component



## What should reviewers focus on?
* Potential Yarn issues caused by removing and regenerating the yarn.lock file


## Checklist
- [Y] My PR name is descriptive and in imperative tense
- [Y] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [Y] I have run the appropriate linter(s)
- [Y] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
